code/modules/organs/internal/lungs/filter.dm

```diff
@@ -19,7 +21,7 @@
 		),
 		"plasma" = list(
 			new /datum/lung_gas/metabolizable("toxins", min_pp=16, max_pp=280),
-			new /datum/lung_gas/waste("oxygen",         max_pp=10),
+			new /datum/lung_gas/waste("carbon_dioxide",         max_pp=10),
```

Comment:
Oh, heck. Sorry. There was a pr to change the gas plasmamen breathed out from oxygen to CO2 so people could make plasmamen rooms but it missed the filter lungs